### PR TITLE
#163198397 Implement user email verification

### PR DESCRIPTION
**Pull request by julietezekwe**
#### What does this PR do?
Implement user email verification

#### Description of Task to be completed?
User clicks on the email received after registration; 
the user will get a success email on email verified and this is updated in the database.

#### How should this be manually tested?
- Signup a new user on this platform, 
- click on the email received after successful signup
- A message that reads email successfully verified will be displayed
- Check the database and see the verified status change from false to true

#### Any background context you want to provide?
Run `npm migrate` before testing this feature.

#### What are the relevant pivotal tracker stories?
[#163198397](https://www.pivotaltracker.com/story/show/163198397)

#### Screenshots (if appropriate)
None

#### Questions:
None

### CODE REVIEW

File: src/helpers/TokenAuthenticate.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ class TokenAuthenticate {
    * @memberof TokenAuthenticate
    */
   static async tokenVerify(req, res, next) {
-    const token = req.headers.Authorization || req.headers['x-access-token'] || req.query.token || req.body.token;
+    const token = req.headers.authorization || req.headers['x-access-token'] || req.query.token || req.body.token;
```
Comment (andela-dbamidele) [inline src/helpers/TokenAuthenticate.js:32]: Why did you change it to lowercase?

Comment (andela-dbamidele) [reply src/helpers/TokenAuthenticate.js:32]: I mean `Authorization`

**Review by andela-dbamidele — CHANGES_REQUESTED** [review]

File: src/helpers/TokenAuthenticate.js (hunk 1 above)
Comment (julietezekwe) [reply src/helpers/TokenAuthenticate.js:32]: I had to change it to the small letter because that is the way the req comes in even when the name is specified as 'Authorization' with capital A in the header. It is converted to small 'a' in the req object. It was throwing up an error for everyone until I discovered that.
